pygeos/coordinates.py

```diff
@@ -136,18 +136,29 @@ def get_coordinates(geometry, include_z=False, return_index=False):
 
 
 def set_coordinates(geometry, coordinates):
-    """Returns a copy of a geometry array with different coordinates.
+    """Sets the coordinates of a geometry array with different coordinates.
 
     If the coordinates array has shape (N, 2), all returned geometries
     will be two-dimensional, and the third dimension will be discarded,
     if present. If the coordinates array has shape (N, 3), the returned
     geometries preserve the dimensionality of the input geometries.
 
+    .. warning::
+
+        The geometries are modified in place! If you don't want to
```

Comment:
Not exactly true, the geometries are not modified inplace, but the *array* is. The geometries in the array are replaced with new ones. So the geometry itself is immutable.

```suggestion
        The geometry array is modified in place! If you don't want to
```

Reply:
Ah, yes, that was indeed incorrectly worded. Thanks for the edits!